subsys/logging/Kconfig.template.log_format_config

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2022 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+choice "LOG_BACKEND_$(backend)_OUTPUT"
+	prompt "Backend Output Mode"
+	default LOG_BACKEND_$(backend)_OUTPUT_TEXT
+
+config LOG_BACKEND_$(backend)_OUTPUT_TEXT
+	bool "Text"
+	help
+	  Output in text.
+
+config LOG_BACKEND_$(backend)_OUTPUT_SYST
+	bool "MIPI SyS-T"
+	depends on LOG_BACKEND_$(backend)
```

Comment:
No need for this, as the if in `Kconfig.backends` already implies this.

Reply:
Okay. Removed this dependency.